models/attendence.js

```diff
@@ -1,37 +1,37 @@
-import mongoose from "mongoose";
-const Schema = mongoose.Schema;
-
-const schema = Schema({
-    time_in: { 
-    type: String, 
-    required: true, 
-    },
-    time_out:{
-    type: String, 
-    required: true,
-    },
-    status: {
-    type: String,
-    enum: ['present', 'absent', 'leave', 'late'],
-    required: true,
-    },
-    date:{
-        type: Date,
-        required: true
-    },
-    updated_date:{
-        type: Date,
-        required: true
-    },
-    remarks:{
-        type: String,
-        maxlength: 250,
-    },
-    monthly_hours:{
-        type: Number,
-        required: true
-    },
-},
-// { timestamps: true },
-);
-export default mongoose.model("Attendence", schema);
+import mongoose from "mongoose";
+const Schema = mongoose.Schema;
+
+const schema = Schema({
+    time_in: { 
+    type: String, 
+    required: true, 
+    },
+    time_out:{
+    type: String, 
+    required: true,
+    },
+    status: {
+    type: String,
+    enum: ['present', 'absent', 'leave', 'late'],
+    required: true,
+    },
+    date:{
+        type: Date,
+        required: true
+    },
+    updated_date:{
+        type: Date,
+        required: true
+    },
+    remarks:{
+        type: String,
+        maxlength: 250,
+    },
+    monthly_hours:{
+        type: Number,
+        required: true
+    },
+},
+// { timestamps: true },
+);
+export default mongoose.model("Attendence", schema);

```

Comment:
Does Attendance schema necessary for otp ?

Reply:
Sir i think it depends on the policy of any organization they're strict on this or not like if they want to make sure that the employee should be physically there then **yes** the OTP will be sent either on the device that's in the organization, at the same time if we're talking that without physical attendance then I really don't think we need to send